helm/botkube/templates/communicationsecret.yaml

```
@@ -1,7 +1,8 @@
+{{- if not .Values.communications.existingSecret -}}
```

```suggestion
{{- if not .Values.communications.existingSecretName -}}
```

How about this?
Right now, if one does following, current logic tries to use the secret even if we have set it to not use it (`false`).
```
 --set communications.existingSecretName="test" --set communications.existingSecret=false
```
With the suggested change we can remove the `existingSecret` value as well.